src/data/auroradb.ts

```diff
@@ -0,0 +1,249 @@
+
+        const queryResult = await executeQuery(queryInput, [nocCode]);
+
+        const resultRow = JSON.parse(JSON.stringify(queryResult));
+
+        return (
+            resultRow?.map((item: ServiceType) => ({
+                lineName: item.lineName,
+                startDate: convertDateFormat(item.startDate),
+                description: item.description,
+            })) || []
+        );
+    } catch (err) {
+        throw new Error(`Could not retrieve services from DynamoDB: ${err.name}, ${err.message}`);
+    }
+};
+
+export const batchGetStopsByAtcoCode = async (atcoCodes: string[]): Promise<Stop[] | []> => {
+    let orQuery = '';
+
+    try {
+        atcoCodes.forEach((code, index) => {
+            orQuery += `atcoCode = ${code} ${index < atcoCodes.length - 1 ? 'OR ' : ''}`;
+        });
+
+        const batchQuery = `Select * from naptanStop where ${orQuery}`;
+
+        const results = await executeQuery(batchQuery, []);
+
+        const parsedResults = JSON.parse(JSON.stringify(results));
+
+        return parsedResults.map((item: NaptanInfo) => ({
+            stopName: item.commonName,
+            naptanCode: item.naptanCode,
+            atcoCode: item.atcoCode,
+            localityCode: item.nptgLocalityCode,
+            localityName: item.localityName,
+            parentLocalityName: item.parentLocalityName !== null ? item.parentLocalityName : '',
+            indicator: item.indicator,
+            street: item.street,
+        }));
+    } catch (error) {
+        console.error(`Error performing batch get for naptan info for stop list: ${atcoCodes}, error: ${error}`);
+        throw new Error(error);
+    }
+};
+
+export const getAtcoCodesByNaptanCodes = async (naptanCodes: string[]): Promise<NaptanAtcoCodes[]> => {
+    const listOfNaptanCodes = naptanCodes.toString();
+    const atcoCodesByNaptanCodeQuery = `Select * from naptanStop where naptanCode in ?`;
+
+    try {
+        const results = await executeQuery(atcoCodesByNaptanCodeQuery, [listOfNaptanCodes]);
+
+        const queryResults = JSON.parse(JSON.stringify(results));
+        return queryResults.map((item: any) => ({ atcoCode: item.atcoCode, naptanCode: item.NaptanCode }));
+    } catch (error) {
+        console.error(
+            `Error performing queries for ATCO Codes using Naptan Codes: ${naptanCodes}. Error: ${error.stack}`,
+        );
+        throw new Error(error);
+    }
+};
+
+export const getServiceByNocCodeAndLineName = async (nocCode: string, lineName: string): Promise<RawService> => {
+    const serviceQuery =
+        'SELECT os.operatorShortName, os.serviceDescription, os.lineName, pl.fromAtcoCode, pl.toAtcoCode, pl.journeyPatternSectionId, pl.order, nsStart.commonName AS fromCommonName, nsStop.commonName as toCommonName ' +
+        ' FROM tndsOperatorService AS os' +
+        ' INNER JOIN tndsJourneyPatternSection AS ps ON ps.operatorServiceId = os.id' +
```

Comment:
The schema has been updated so the table is now `tndsJourneyPattern` instead of `tndsJourneyPatternSection` now